packages/core/src/run-engine-app.ts

```diff
@@ -55,10 +55,8 @@ export async function runEngineApp({
     ]);
     const [runningFeature] = allFeatures;
 
-    const engine = new RuntimeEngine([COM.use({ config: { resolvedContexts, publicPath } }), ...config], options).run(
-        runningFeature,
-        envName
-    );
+    const engine = new RuntimeEngine([COM.use({ config: { resolvedContexts, publicPath } }), ...config], options);
+    await engine.run(runningFeature, envName);
```

Comment:
I am not sure we want to await here.. the await should be done in the higher level, as discussed with @barak007

Reply:
there is no higher level. runEngineApp is a part of the public API of engine-core.